src/main/java/com/moodybluez/enterprise/controller/MoodController.java

```diff
@@ -22,12 +22,8 @@ Mood getById(@PathVariable int id) {
 
     @GetMapping("/mood")
     List<Mood> get() {
-        List<Mood> ret = new ArrayList<>();
         Map<Integer, Mood> entities =  moodService.fetchAll();
-        for(Mood mood:entities.values()){
-            ret.add(mood);
-        }
-        return ret;
+        return new ArrayList<>(entities.values());
```

Comment:
This is unnecessary addition of "lines of code".  If all you want to do is create Collection of values from another without modifying any value(s), you can just pass the collection in the constructor call when creating new ArrayList. This reduces lines of code and additional steps that program has to compile and run.